### PR TITLE
LiveMap

### PR DESCRIPTION
**Pull request by cixio**

### CODE REVIEW

**Comment by dl9rdz** [comment]
Ok, let's just merge it, code seems fine for me.

A minor issue (for me on my old Macos Mojave) ist that the yellow circle is a Unicode 12 sysmbol from 2019, not showing on older systems.

I also modified the that bar to make it fit in a single line, but now that is too crowded on my small smartphone display. Probably needs some other solution to be perfect.

File: RX_FSK/data/livemap.js [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+          setTimeout(function() {$('#status').html('🟡');},100);
+        }
+        if (data.gps) {
+          gps(data.gps);
+        }
+      });
+  };
+
+  predictor = false;
+  get_predict = function(data) {
```
Comment (LukePrior) [inline RX_FSK/data/livemap.js:223]: Hey, do you know if the latitude is in the range (0 - 180) or (-90 to +90)?

I'm just checking in reference to #142

Comment (cixio) [reply RX_FSK/data/livemap.js:223]: Source of this is:

`SondeInfo *s = &sonde.sondeList[sonde.currentSonde];`

I think @dl9rdz knows better than me.

Comment (dl9rdz) [reply RX_FSK/data/livemap.js:223]: I guess this is a longitude issue, not latitude.

My software uses -180 to 180 and so does leaflet, but the tawhiri API uses 0 - 360. I noted that problem with my Android App earlier this year when I was on Iceland. I guess we will need a similar fix for the livemap
https://github.com/dl9rdz/rdzwx-go/commit/0fe5f00751671c8bdd4fa1f9d534b7c24f3378d4

Comment (cixio) [reply RX_FSK/data/livemap.js:223]: Fixed, see pull request: https://github.com/dl9rdz/rdz_ttgo_sonde/pull/148

**Pull request merged**